go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/openshift/cluster-ingress-operator
 
 go 1.21
 
-toolchain go1.21.5
+toolchain go1.21.7
```

Comment:
Did you mean to include this change in your PR?

Reply:
Yes, this will jump depending on the env for the creator of the PR. Removed it, jut to reduce the diff.